libs/island-ui/core/src/lib/Tooltip/Tooltip.tsx

```diff
@@ -107,7 +108,7 @@ export const Tooltip: FC<TooltipProps> = ({
           <TooltipArrow {...tooltip}>
             <ArrowIcon placement={tooltip.placement} />
           </TooltipArrow>
-          {text}
+          {HtmlParser(text)}
```

Comment:
i am wondering if you should use Markdown instead? I dont mind this, this is more flexible, we were just starting to use Markdown to handle cases like this

Reply:
I started using Markdown, but was super buggy and didn't work for this. Used HtmlParser as the fallback and worked far better.

Reply:
same concerns here as in the radio button regarding the parser, and maybe to add to that, it's somewhat hidden to the developer that the string can be html